heron/tools/tracker/src/python/routers/container.py

```diff
@@ -186,29 +189,30 @@ async def _get_exception_log_response(
 @router.get("/exceptions", response_model=List[ExceptionLog])
 async def get_exceptions(  # pylint: disable=too-many-arguments
     cluster: str,
-    role: Optional[str],
     environ: str,
     component: str,
-    instances: List[str] = Query(..., alias="instance"),
+    instances: Optional[List[str]] = Query(None, alias="instance"),
     topology_name: str = Query(..., alias="topology"),
+    role: Optional[str] = None,
 ):
   """Return info about exceptions that have occurred per instance."""
   exception_response = await _get_exception_log_response(
       cluster, role, environ, component, instances, topology_name, summary=False
   )
-
-  return [
-      ExceptionLog(
-          hostname=exception_log.hostname,
-          instance_id=exception_log.instance_id,
-          stack_trace=exception_log.stacktrace,
-          lasttime=exception_log.lasttime,
-          firsttime=exception_log.firsttime,
-          count=str(exception_log.count),
-          logging=exception_log.logging,
-      )
-      for exception_log in exception_response.exceptions
-  ]
+  print(f"NICK: exception response: {exception_response}")
+
+  ret = []
+  for exception_log in exception_response.exceptions:
+    ret.append(ExceptionLog(
+      hostname = exception_log.hostname,
+      instance_id = exception_log.instance_id,
+      stacktrace = exception_log.stacktrace,
+      lasttime = exception_log.lasttime,
+      firsttime = exception_log.firsttime,
+      count = str(exception_log.count),
+      logging = exception_log.logging,
+    ))
```

Comment:
Indentation and whitespace fix

```suggestion
    ret.append(ExceptionLog(
        hostname=exception_log.hostname,
        instance_id=exception_log.instance_id,
        stacktrace=exception_log.stacktrace,
        lasttime=exception_log.lasttime,
        firsttime=exception_log.firsttime,
        count=str(exception_log.count),
        logging=exception_log.logging,
    ))
```